src/components/Navbar.js

```diff
@@ -0,0 +1,46 @@
+import { NavLink, useLocation } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import AddLink from './AddLink'
+import medicationsDisabled from '../icons/pill-disabled.svg'
+import medicationsEnabled from '../icons/pill-enabled.svg'
+import observationsDisabled from '../icons/observations-disabled.svg'
+import observationsEnabled from '../icons/observations-enabled.svg'
+
+export default function Navbar() {
+  const location = useLocation()
+
+  return (
+    <Nav>
+      <NavLinkStyled exact to="/medications">
+        {location.pathname.includes('medications') ? (
+          <img src={medicationsEnabled} alt="" height="32px" />
+        ) : (
+          <img src={medicationsDisabled} alt="" height="32px" />
+        )}
+      </NavLinkStyled>
+      <NavLinkStyled exact to="/observations">
+        {location.pathname.includes('observations') ? (
+          <img src={observationsEnabled} alt="" height="32px" />
+        ) : (
+          <img src={observationsDisabled} alt="" height="32px" />
+        )}
+      </NavLinkStyled>
```

Comment:
It seems like this could be refactored quite a bit. Also, here I'd set the `alt` attribute since the Link doesn't have any text, right? Just to make it more accessible. And if you set the `height` like this, you'd want to set the `width` as well.